src/App.tsx

```diff
@@ -1,14 +1,80 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodosFilters } from './types/TodosFilters';
+import { Filter } from './types/Filter';
+
+function getFilteredTodos(todos: Todo[], filter: Filter): Todo[] {
+  let filteredTodos = todos.filter(todo => {
```

Comment:
If filter is all `TodosFilters.All` you don't to filter at all

Reply:
Not fixed, you can add `if` for handling that